docs/users/index.md

```diff
@@ -25,15 +25,15 @@ BetterDiscord fully integrates Twitch emotes sourced from BTTV, Twitch, and FFZ.
 
 ### Live Editor
 
-BetterDiscord allows you to modify your Discord application and see the changes apply in real time.
+BetterDiscord allows you to modify your Discord application and see the changes apply in real-time.
 
 ### Transparency
 
 BetterDiscord can make the Discord application partially see-through leading to very cool designs and desktop environments.
 
 ### Public Servers
```

Comment:
Shouldn't this whole block be removed instead?

Reply:
Im gonna assume you are talking about the public servers part, and it is now removed. Sorry about missing that.